apps/kg/modules/administration/title-information/index.tsx

```diff
@@ -1,29 +1,34 @@
-import { useAdministrationStatus } from "../hooks";
+import { useGetAllAdministration } from "../hooks";
 import React, { FC, ReactElement } from "react";
 import { RiErrorWarningFill } from "react-icons/ri";
 
 export const TitleAdministration: FC = (): ReactElement => {
-  const { getAdministrationStatus } = useAdministrationStatus();
+  const { data } = useGetAllAdministration();
+  const getAdministrationStatus = data?.data;
 
   return (
     <div className="title lg:px-20 px-5 w-full">
       <h1 className="text-[20px] font-[600]  text-black md:text-left mb-7">Administrasi</h1>
       <div
         className={`${
-          getAdministrationStatus === "none" ? "bg-yellow-100" : "bg-primary-100"
+          getAdministrationStatus?.status === "NOT-SUBMITTED" ? "bg-yellow-100" : "bg-primary-100"
```

Comment:
status status gini jadiin enum lit

gini contoh nya

export enum ADMINISTRATION_STATUS {
  NOT = "NOT-SUBMITED",
  PENDING = "PENDING"
}

nanti di simpen di file enum.ts

maksudnya adalah kalau saja kedepan nya ada perubahan type ngedit nya cuma sekali di enum.ts aja